ci/src/cI_target.ml

```diff
@@ -87,6 +87,10 @@ let path = function
 
 type v = [ `PR of PR.t | `Ref of Ref.t ]
 
+let pp_v f = function
+  | `PR pr  -> Fmt.pf f "pr/%a" PR.pp pr
+  | `Ref x -> Fmt.pf f "refs/%a" Ref.pp x
```

Comment:
I'm only using this pp_v function for branch name calculation, so the values don't seem significant (but should be consistent). I'll modify it to be plural to match Git then.